src/main/scala/org/allenai/deep_qa/data/Instance.scala

```diff
@@ -152,3 +152,32 @@ case class McQuestionAnswerInstance(
     Seq(Seq(s"$passage\t$question\t$answerString" + labelString))
   }
 }
+
+trait BaseTaggingInstance extends Instance
```

Comment:
It's not required, it just gives us a more specific type to return for a sequence tagging dataset reader.  So, e.g., if you want to enumerate all possibilities with a `match ... case` statement, you have many fewer to worry about.